test/functional/uri_options_spec_tests.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const fs = require('fs');
```

Comment:
Unrelated: we do this a lot. We should make a `loadSpecTests` function and reuse it where possible.